src/data/markdown/translated-guides/en/02 Using k6/06 Test life cycle.md

```diff
@@ -1,10 +1,15 @@
 ---
 title: 'Test life cycle'
-excerpt: 'The four distinct life cycle stages in a k6 test are "init", "setup", "VU" and "teardown".'
+excerpt: 'The four distinct life cycle stages in a k6 test are "init", "setup", "VU", and "teardown".'
 ---
 
-The four distinct life cycle stages in a k6 test are "init", "setup", "VU" and "teardown"
-Throughout the documentation, you will also see us referring to it as "init code", "VU code" etc.
+A k6 test has four distinct stages, which always run in the same order:
+
+1. *Init code* initializes VUs
+2. *Setup code* sets up data and the test environment
```

Comment:
might be good to mark it as Optional.

Reply:
have doubts. 

2. *Setup code* sets up data and the test environment. Optional. 

Don't hold a strong opinion.